src/ImGui.NET/ImGuiNative.cs

```diff
@@ -738,12 +735,12 @@ internal static void igPushFont(object nativeFont)
         // public state access - if you want to share ImGui state between modules (e.g. DLL) or allocate it yourself
         [DllImport(cimguiLib)]
         public static extern string igGetVersion();
-        [DllImport(cimguiLib)]
-        public static extern void* igGetpublicState();
-        [DllImport(cimguiLib)]
-        public static extern uint igGetpublicStateSize();
-        [DllImport(cimguiLib)]
-        public static extern void igSetpublicState(void* state, bool construct);
+        /*
```

Comment:
I haven't really looked at them, as I don't have any use for them yet (maybe never). Maybe they could be examined later if needed?